sdks/python/apache_beam/runners/interactive/utils_test.py

```diff
@@ -56,7 +56,7 @@ def test_parse_windowedvalue_with_window_info(self):
         [['a', 2, int(1e6), els[0].windows, els[0].pane_info],
          ['b', 3, int(1e6), els[1].windows, els[1].pane_info]],
         columns=[0, 1, 'event_time', 'windows', 'pane_info'])
-    pd.testing.assert_frame_equal(actual_df, expected_df)
+    pd.testing.assert_frame_equal(actual_df, expected_df, check_like=True)
```

Comment:
This test was flaky because the dataframe columns can be built in arbitrary orders. This option makes sure it doesn't take column positioning into consideration since we only care about the equivalence of data.